BuildScript.UnitTests/CreateTestConfigurationsTest.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.  You may obtain a copy of the
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using NUnit.Framework;
+using Remotion.BuildScript.BuildTasks;
+
+namespace BuildScript.UnitTests
+{
+  [TestFixture]
+  public class CreateTestConfigurationsTest
+  {
+    [Test]
+    public void CreatedTestConfigurations_ParsedCorrectly ()
+    {
+      var task = new CreateTestConfigurations
+                 {
+                     SupportedBrowsers = new ITaskItem[] { new TaskItem ("Chrome"), new TaskItem ("Firefox") },
+                     EnabledBrowsers = new ITaskItem[] { new TaskItem ("Chrome"), new TaskItem ("NoBrowser") },
+                     SupportedDatabaseSystems = new ITaskItem[] { new TaskItem ("SqlServer2012"), new TaskItem ("SqlServer2014") },
+                     EnabledDatabaseSystems = new ITaskItem[] { new TaskItem ("SqlServer2012"), new TaskItem ("NoDB") },
+                     SupportedPlatforms = new ITaskItem[] { new TaskItem ("x64") },
+                     EnabledPlatforms = new ITaskItem[] { new TaskItem ("x64") },
+                     SupportedExecutionRuntimes =
+                         new ITaskItem[] { new TaskItem ("Win_NET45::DockerImageForWinNet45"), new TaskItem ("Win_NET46::DockerImageForWinNet46") },
```

Comment:
Yeah I was thinking this would come up in a review. The single ":" was taken by Docker, ";" was taken by MSBuild, I'm not sure what else would make a good separation character, maybe "|"?

Reply:
We ended up choosing `=` as the separation character